packages/core/README.md

```diff
@@ -32,11 +32,12 @@ Since each step runs in a separate process, you can use `exportVariable` to add
 core.exportVariable('envVar', 'Val');
 ```
 
-#### Setting a secret
+#### Setting a string as secret
```

Comment:
personally i prefer the existing verbiage :)

Reply:
..just my 2 cents. i'll defer to others

Reply:
I agree.